fuzz/fuzz_targets/lerp.rs

```diff
@@ -0,0 +1,32 @@
+use honggfuzz::fuzz;
```

Comment:
Can you explain a bit what's going on here? In general we want to avoid non-determinism as it makes reproducing bugs much harder (the same random number won't be used and the bug may not reproduce. I do see that you seed the random number from the test case itself, so this may be ok, but would like to hear if this is guaranteed in StdRng (see https://stackoverflow.com/questions/62309424/does-numpy-random-seed-always-give-the-same-random-number-every-time for commentss).

Reply:
StdRng is guaranteed to generate the same random number when ran on the same build/platform. I'll swap this to Pcg64 to ensure that it generates the same random number on all builds/platforms.